scripts/mockGuardianSpy.ts

```diff
@@ -0,0 +1,347 @@
+import {
+    Server,
+    ServerCredentials,
+} from '@grpc/grpc-js';
+import {MockGuardians} from "@certusone/wormhole-sdk/lib/cjs/mock";
+const GUARDIAN_PRIVATE_KEY = '0x5756c5018454b83225f8e8264651c3a8032c4ac931e1d310d81239754f1bb791';
+const guardians = new MockGuardians(0, [
+    GUARDIAN_PRIVATE_KEY,
+  ]);
+
+const RetryProvider = require("../packages/package-utils").RetryProvider;
```

Comment:
Didn't I see this imported via `index` elsewhere? Can we be consistent?

Reply:
Made this and the instance of `index` being used (which is in `packages/reputation-miner/ReputationMiner.js` consistent (but both have changed 😛 )